### ISSUE TITLE
Export markups planes

### ISSUE THREAD

**Issue by mauigna06** [PerkLab/SlicerOpenAnatomy#10]
It would be nice if planes could be exported as they usually used in surgical planning to mark surgical margins

**Comment by lassoan** [comment]
Yes, it would be useful to export all markup types (planes, lines, point sets, etc.). It would be great if you could implement it. For example, you could create a temporary model node from the markup node [here](https://github.com/PerkLab/SlicerOpenAnatomy/blob/8856b1e5f6eceee7be90697857f6cf90dbace50f/OpenAnatomyExport/OpenAnatomyExport.py#L352).

**Comment by mauigna06** [comment]
I didn't find a trivial way to do this. A plane-markup rendering code is distributed on different actors, mappers and props.
Do you have any tip on how to convert a (plane) markup to a model? In particular I would like to get the nice translucent opacity of the body of the planeNodes and their nice opaque outline with their corresponding colors

**Comment by lassoan** [comment]
You can create a plane mesh using vtkPlaneSource. You can get the corners by transforming the [+/-halfSize, +/-halfSize, 0, 1] point from the object coordinate system to world coordinate system by multiplying by [GetObjectToWorldMatrix](https://apidocs.slicer.org/master/classvtkMRMLMarkupsPlaneNode.html#a3eb521ad797cdf490727eb4309bd60e4).

**Comment by mauigna06** [comment]
Pull request opened: https://github.com/PerkLab/SlicerOpenAnatomy/pull/12
